lib/core-db-pg/queries.ts

```diff
@@ -207,15 +208,42 @@ export const getVestingTotalForAddress = async (_address: string) => {
   }
 };
 
-export const getUnlockedSupply = async () => {
-  const latestBlock = await getLatestBlock();
-  const sql = 'SELECT sum(vesting_value::bigint) FROM account_vesting where block_id < $1;';
+export interface UnlockedSupply {
+  blockHeight: string;
+  unlockedSupply: BigNumber;
+}
+
+export async function getUnlockedSupply(): Promise<UnlockedSupply> {
+  const sql = `
+    WITH 
+    block_height AS (SELECT MAX(block_id) from accounts),
+    totals AS (
+      SELECT DISTINCT ON (address) credit_value, debit_value 
+        FROM accounts 
+        WHERE type = 'STACKS' 
+        AND address !~ '(-|_)' 
+        AND length(address) BETWEEN 33 AND 34 
```

Comment:
I would like to get confirmation if this is valid from someone like Jude - I suspect it is because your result matches his.